docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
 
   ipfs.node.2:
     container_name: "ipfs_node_2"
-    image: ipfs/go-ipfs:v0.4.19
+    image: ipfs/go-ipfs:v0.4.20
     command: daemon --migrate=true --enable-pubsub-experiment
```

Comment:
might want to change this to use `--enable-namesys-pubsub` which enables both pubsub, and ipns pubsub. In theory in might make our tests quicker